src/components/close-button/index.tsx

```diff
@@ -0,0 +1,23 @@
+import {h} from 'preact';
+import * as styles from './close-button.scss';
+import {A11yWrapper} from '../a11y-wrapper/a11y-wrapper';
+import {icons} from '../icons';
+const {Icon} = KalturaPlayer.ui.components;
+
+interface CloseButtonProps {
+  onClick: () => void;
+}
+
+export const CloseButton = (props: CloseButtonProps) => (
+  <A11yWrapper onClick={props.onClick}>
+    <button className={[styles.closeButton, 'kaltura-navigation__close-button'].join(' ')} style={{'top': '20px', 'right': '10px'}} tabIndex={0}>
```

Comment:
why you add inline styles to overwrite https://github.com/kaltura/playkit-js-navigation/pull/263/files#diff-f3517f9f6c2c05ffb18eeace9eda1f6fb48cd91cb0978348d4e734a42583756fR3-R4 ?